docs/support/tutorials/python-usage-guide.md

```diff
@@ -0,0 +1,291 @@
+# Using Python on CSC supercomputers
+
+Some important aspects of working with the Python programming language
+are notably different on CSC supercomputers compared to usage on a personal
+device or in other HPC environments. To make the most of the computational
+resources available to you, it is helpful to be aware of the differences.
+
+See the
+[Python application page](../../apps/python.md)
+for general information on the Python language and
+pre-installed Python environments on CSC supercomputers.
+
+## Creating and managing Python environments
+
+### Installing Python packages to existing modules
+
+If there is a CSC-provided module that covers _almost_ everything you
+need, but it is missing a few Python packages, you can try installing
+those yourself with the pip package manager.
+
+See the [package lists on our Python application
+page](../../apps/python.md#pre-installed-python-environments) to find out
```

Comment:
The test scripts in _tests/_ are based on _grep_, so they don't see links that span multiple lines. My suggestion (for now) is to disregard any line length limits when it comes to links.

Reply:
All links are now single-line